daemon/k8s_watcher.go

```diff
@@ -996,51 +953,10 @@ func (d *Daemon) k8sServiceHandler() {
 				scopedLog.WithError(err).Error("Unable to add/update service to implement k8s event")
```

Comment:
Could you add a comment to this section explaining that we manage the lbmap entries here (I think?), including updates, but that we create toCIDR equivalents in cilium-operator. I think it will be really unclear where the data is coming from in some cases.